DQMOffline/Trigger/python/HigPhotonJetHLTOfflineSource_cfi.py

```diff
@@ -28,7 +28,7 @@
     dirname = cms.untracked.string("HLT/Higgs/PhotonJet"), 
     verbose = cms.untracked.bool(False), # default: False
     triggerAccept = cms.untracked.bool(True), # default: True 
-    triggerResultsToken = cms.InputTag("TriggerResults"),
+    triggerResultsToken = cms.InputTag("TriggerResults","","HLT"),
```

Comment:
Also, I looked at other DQM related modules and they all explicitly were using "HLT" to get the HLT trigger results.